src/bundle-cli.ts

```diff
@@ -10,57 +10,70 @@ import { argv } from "./arguments";
-        ProjectDirectory: path.resolve(process.cwd(), argumentValues.project)
-    };
+function argumentsToConfig(
+  argumentValues: Contracts.ArgumentsValues
+): Contracts.Config {
+  return {
+    Destination: argumentValues.dest,
+    Entry: argumentValues.entry,
+    DedupeGlobs: argumentValues.dedupe,
+    Verbosity: resolveVerbosity(argumentValues.verbosity),
+    IncludePaths: argumentValues.includePaths,
+    IgnoredImports: argumentValues.ignoredImports,
+    ProjectDirectory: path.resolve(process.cwd(), argumentValues.project)
+  };
 }
 
 async function main(argumentValues: Contracts.ArgumentsValues): Promise<void> {
-    const config = argumentsToConfig(argumentValues);
-    const isWatching = argumentValues.watch != null;
-    const bundler = new Launcher(config);
+  const config = argumentsToConfig(argumentValues);
+  const isWatching = argumentValues.watch !== null;
+  const noWatch = argumentValues.watch === "";
+  const bundler = new Launcher(config);
 
-    if (argumentValues.verbosity !== Contracts.Verbosity.None && (argumentValues.entry == null || argumentValues.dest == null)) {
-        console.error("[Error] 'entry' and 'dest' are required.");
-        process.exit(1);
-    }
+  if (
+    argumentValues.verbosity !== Contracts.Verbosity.None &&
+    (argumentValues.entry == null || argumentValues.dest == null)
+  ) {
+    console.error("[Error] 'entry' and 'dest' are required.");
+    process.exit(1);
+  }
 
-    if (argumentValues.verbosity !== Contracts.Verbosity.None && isWatching && argumentValues.watch === "") {
-        console.error("[Error] 'watch' must be defined.");
-        process.exit(1);
-    }
+  if (
+    !noWatch &&
+    isWatching &&
+    argumentValues.verbosity !== Contracts.Verbosity.None
+  ) {
+    console.error("[Error] 'watch' must be defined.");
+    process.exit(1);
+  }
 
-    if (isWatching) {
-        const onFilesChange = debounce(async () => {
-            if (config.Verbosity === Contracts.Verbosity.Verbose) {
-                console.info("[Watcher] File change detected.");
-            }
-            await bundler.Bundle();
-            if (config.Verbosity === Contracts.Verbosity.Verbose) {
-                console.info("[Watcher] Waiting for changes...");
-            }
-        }, 500);
+  if (!noWatch && isWatching && !Contracts.Verbosity.None) {
+    const onFilesChange = debounce(async () => {
+      if (config.Verbosity === Contracts.Verbosity.Verbose) {
+        console.info("[Watcher] Waiting for changes...");
+      }
+      await bundler.Bundle();
+    }, 500);
+    chokidar.watch(argumentValues.watch).on("change", onFilesChange);
+  }
 
-        chokidar.watch(argumentValues.watch).on("change", onFilesChange);
-    }
+  await bundler.Bundle();
 
-    await bundler.Bundle();
-    if (isWatching && config.Verbosity === Contracts.Verbosity.Verbose) {
-        console.info("[Watcher] Waiting for changes...");
-    }
+  if (
+    !noWatch &&

```

Comment:
This added logic checks for noWatch,  then exits the process if conditions aren't met...otherwise, logs watch message to the console and keeps the process open